doc/design/linc_permissions.md

```diff
@@ -0,0 +1,54 @@
+# Permissions for S3 Assets in LINC Archive
+
+Collaborators: Aaron Kanzer, Kabilar Gunalan
+
+## Requirements
+
+• As a CONNECTS developer, I shall be able to view multiple datasets/layers (e.g. dMRI, tractography, HiP-CT) in the same neuroglancer
+• As a CONNECTS developer, I shall be able to view data locally or on neuroglancer
+• As a CONNECTS developer, I shall be able to have direct access to S3 asset URI
+• As a LINC Archive user, I shall be able to render neuroglancer layers view similar to DANDI
+
+#### Relevant Links
+
+https://github.com/orgs/lincbrain/projects/2?pane=issue&itemId=54650571
+
+https://github.com/neuroscales/ngtools/blob/main/notebooks/show_tract.ipynb
+
+https://github.com/google/neuroglancer/issues/507
+
+## Implementation Options
+
+#### Issue read-only credentials for S3 bucket. Quick set up.
+
+Does not resolve rendering issue. Would resolve accessing the asset in Jupyter Notebook, but would require flaky Python code.
+
+#### Integrate read-only credentials for S3 bucket into LINC Archive, LINC CLI
+
+###### CloudFront distribution with Origin Access Identity control for relevant S3 buckets
+
+- S3 Asset Bucket is served via a CloudFront distribution
+- User hits a get_presigned_cookie/ API endpoint in LINC Archive. In the endpoint response is a valid cookie that allows the user's browser to be able to access the asset via their browser
+
+Rendering via Neuroglancer
+
+- (Might need to modify fork of neuroglancer to handle CloudFront asset path in same format that it handles S3 asset path)
+
```

Comment:
```suggestion
- Host Neuroglancer fork on S3 bucket, and make available at `neuroglancer.lincbrain.org`.

```

Reply:
```
Host Neuroglancer fork on S3 bucket, and make available at `neuroglancer.lincbrain.org`.
```

This perhaps should maybe read as:

```
Host Neuroglancer fork on S3 bucket with static site hosting enabled. Create an AWS CloudFront distribution to sit as a verified proxy for permission to many S3 assets at once via presigned cookies.  Provide CNAME for CloudFront Distribution URL to `neuroglancer.lincbrain.org` so that cookies generated by LINC Archive (also living at a `*.lincbrain.org` can be shared between the API and the Neuroglancer site.
```

Reply:
Thanks Aaron.  I have updated by suggestion according to your feedback.